src/components/utils/test/date_time_test.cc

```diff
@@ -108,11 +108,10 @@ TEST(DateTimeTest, CalculateTimeSpan) {
   // arrange
   const TimevalStruct time = date_time::DateTime::getCurrentTime();
 
-  const uint32_t sleep_time_mSec = 10;
+  const uint64_t sleep_time_mSec = 100;
```

Comment:
@OHerasym Well, if this is the case my advice is - do not dig in such know-how silently in the code. Please, add at least comment. My other advice is if you found this - what about ticket/proposal SDL to be reviewed is there sleep below 15ms and to be discussed what can be done etc. But at least add a comment here why 100 was choosen.